### PR TITLE
ovirt_host: Add refresh

### PR DESCRIPTION
**Pull request by mnecas**
Fixes https://github.com/oVirt/ovirt-ansible-collection/discussions/672

### CODE REVIEW

File: plugins/modules/ovirt_host.py [hunk 1]
```diff
@@ -31,9 +31,11 @@
         description:
             - "State which should a host to be in after successful completion."
             - "I(iscsilogin) and I(iscsidiscover) are supported since version 2.4."
+            - "I(refresh) is supported since 3.1.0"
         choices: [
             'present', 'absent', 'maintenance', 'upgraded', 'started',
-            'restarted', 'stopped', 'reinstalled', 'iscsidiscover', 'iscsilogin'
+            'restarted', 'stopped', 'reinstalled', 'iscsidiscover', 'iscsilogin',
+            'refresh'
```
Comment (mwperina) [inline plugins/modules/ovirt_host.py:38]: `refreshed` to align with other states?

**Review by mwperina — CHANGES_REQUESTED** [review]
Please add changelog

**Comment by mnecas** [comment]
The CI fails on some ansible-lint pip error

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**